sys-apps/baselayout/baselayout-9999.ebuild

```diff
@@ -9,7 +9,7 @@ CROS_WORKON_REPO="git://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="2b8e4de42771033072324713ba8919248585d263"  # flatcar-master
+	CROS_WORKON_COMMIT="1d32bea2c0e4335d4a8f7e0ccd6a7b41da15e4a7"  # flatcar-master
```

Comment:
I've started to point this merge commit after Kai suggestion. In this case, that would be https://github.com/flatcar-linux/baselayout/commit/b67c163348c69b7c02dac1bb087b1a15cf21afa7

Reply:
Good catch, will update.